Asterix/utils/save_and_read.py

```diff
@@ -288,3 +289,14 @@ def get_data_dir(env_var_name="ASTERIX_DATA_PATH", config_in=None, datadir="aste
         if not os.path.isdir(home_path):
             os.mkdir(home_path)
         return home_path
+
+
+def create_experiment_dir(append=''):
```

Comment:
need to add docstring ! 😱

Reply:
Also I missed it during last PR but get_data_dir docstring is not complete (missing parameters and missing 'default value' for conditional argument)

Reply:
Will fix this, thanks for pointing it out!